### ISSUE TITLE
blueprint based on file exists

### ISSUE THREAD

**Issue by genofire** [goauthentik/authentik#5300]
**Is your feature request related to a problem? Please describe.**
A option to define a state of resources by an blueprint. And not like current define a change/migration.

**Describe the solution you'd like**
That if file is detected the resources are created / updated (like `state: present`) and remove of the file would delete resources (like `state: absent`) with an new state type like `state: watch`

**Describe alternatives you've considered**
easy way to remove based on an blueprint yaml file (without change of file).

**Additional context**
it is for me like ansible vs. terraform ....

**Comment by benedikt-bartscher** [comment]
I really like this approach, i prevents from accidential blueprint deletion if something goes wrong (for example you are missing a volume mount in k8)

**Comment by genofire** [comment]
- a mount happens bevor the container starts in k8s
- after the blueprints are there (again), it will create resources based on blueprint again

**Comment by benedikt-bartscher** [comment]
Yes i know, thats the current behaviour. but with declarative blueprints (this issue) you could prevent the outage completly, even if the blueprints get lost (state: present).
But you can also intentionally delete blueprints (state: absent) or keep the current watch behaviour (state: watch). That is why i like your approach.
